node-src/tasks/upload.ts

```diff
@@ -142,6 +143,9 @@ export const traceChangedFiles = async (ctx: Context, task: Task) => {
     }
 
     const stats = await readStatsFile(statsPath);
+
+    checkStorybookBaseDir(ctx.options.storybookBaseDir, stats);
```

Comment:
I couldn't decide between doing the checking AND the error throwing in this function, or checking in this function and having it return a falsy value and then throwing the error here in upload. Which one is a more consistent pattern in this repo?

Reply:
Yes, `INVALID_OPTIONS` (254) would work here.